### PR TITLE
Replace "BFT nodes" with "Full nodes" in devnet

### PR DESCRIPTION
**Pull request by dlachaume**
## Content
This PR includes some refactoring in the devnet.
BFT nodes are no longer used, replaced by Full nodes. The documentation and variable names have been adapted.

## Pre-submit checklist

- Branch
  - [ ] Tests are provided (if possible)
  - [ ] Crates versions are updated (if relevant)
  - [ ] CHANGELOG file is updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [x] Update README file (if relevant)
  - [x] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

### CODE REVIEW

File: mithril-test-lab/mithril-devnet/README.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ artifacts
 ├── cardano-node
 ├── docker-compose.yaml
 ├── log.sh
-├── node-bft1
+├── node-pool1
```
Comment (jpraynaud) [inline mithril-test-lab/mithril-devnet/README.md:172]: There should be a `node-full1` folder also?

File: mithril-test-lab/mithril-devnet/mkfiles/mkfiles-topology.sh [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
-BFT_NODES=()
-BFT_NODES_N=()
-for (( i=1; i<=${NUM_BFT_NODES}; i++ ))
+FULL_NODES=()
+FULL_NODES_N=()
+for (( i=1; i<=${NUM_FULL_NODES}; i++ ))
   do
-    BFT_NODES=("${BFT_NODES[@]}" "node-bft${i}")
-    BFT_NODES_N=("${BFT_NODES_N[@]}" "${i}")
+    FULL_NODES=("${FULL_NODES[@]}" "full-node${i}")
```
Comment (jpraynaud) [inline mithril-test-lab/mithril-devnet/mkfiles/mkfiles-topology.sh:9]:
```suggestion
    FULL_NODES=("${FULL_NODES[@]}" "node-full${i}")
```

File: mithril-test-lab/mithril-devnet/README.md (hunk 1 above)
Comment (dlachaume) [reply mithril-test-lab/mithril-devnet/README.md:172]: With the default configuration, there are `node-pool1`, `node-pool2` and `node-pool3` directories.
Maybe there's something wrong in the script. I'm gonna investigate 👀

**Review by jpraynaud — APPROVED** [review]
LGTM :+1:

**Review by Alenar — APPROVED** [review]

**Pull request merged**